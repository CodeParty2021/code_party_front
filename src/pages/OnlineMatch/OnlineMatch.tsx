import React from "react";

type Props = {};

export const OnlineMatch: React.FC<Props> = () => {
  return (
    <div>
      <div>オンライン対戦開発予定地</div>
    </div>
  );
};
