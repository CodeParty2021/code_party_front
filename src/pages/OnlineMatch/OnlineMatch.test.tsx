import React from "react";
import { shallow } from "enzyme";
import { OnlineMatch } from "./OnlineMatch";
describe("<OnlineMatch />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<OnlineMatch />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
