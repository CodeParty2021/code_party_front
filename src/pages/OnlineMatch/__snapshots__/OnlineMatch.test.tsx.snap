// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OnlineMatch /> auth snapshot test 1`] = `
Array [
  <div>
    <div>
      オンライン対戦開発予定地
    </div>
  </div>,
]
`;
